examples/docs_snippets/pyproject.toml

```diff
@@ -26,7 +28,7 @@ extend-ignore = [
   "TCH",
 ]
 
-[tool.ruff.isort]
+[tool.ruff.lint.isort]
```

Comment:
Linter settings are in general now under `tool.ruff.lint`, with settings that are shared between Ruff's linter and formatter at the top-level (`tool.ruff`). So this is all just moving settings around between those sections as appropriate.